components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -292,6 +303,7 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                 <a target="_parent" href={this.state.workspace?.contextURL}><p className="w-56 truncate hover:text-blue-600 dark:hover:text-blue-400" >{this.state.workspace?.contextURL}</p></a>
               </div>
             </div>
+            {installationSteps}
             <div className="mt-10 justify-center flex space-x-2">
```

Comment:
I missed the following comment while in draft. 😅 

**issue(non-blocking)**: Probably not the best line to comment this, but needed a place to jot down this minor issue. When you dismiss the alert with _Cancel_ and try clicking the primary action (Open in VS Code on Desktop) the user is redirected to a new page instead of triggering the dialog again.

Reply:
Only this fix just made the day. 🌤️